state-storage/lib/state-storage.ts

```diff
@@ -3,6 +3,20 @@ import {Logger} from "@sailplane/logger";
 
 const logger = new Logger('state-storage');
 
+interface StateStorageOptions {
+    /** If true, do not log values. */
+    quiet?: boolean;
```

Comment:
nit: Would the "verbose" keyword be more familiar and intuitive to use?  It's a logical inversion and we would need to make some code changes to accompany the name change, but I'm mentioning it anyway to confirm this is the direction we want to steer.

Reply:
It looks like a legacy parameter, so... don't mind mind me.